### PR TITLE
Add contains as a trait, and implementing it for relevant primitives

### PR DESCRIPTION
**Pull request by Rhuagh**

### CODE REVIEW

File: src/aabb.rs [hunk 1]
```diff
@@ -243,6 +229,33 @@ impl<S: BaseNum> Aabb for Aabb2<S> {
     }
 }
 
+impl<S: BaseNum> Containment<Point2<S>> for Aabb2<S> {
```
Comment (kvark) [inline src/aabb.rs]: maybe, `Container`?

Comment (vitvakatu) [reply src/aabb.rs]: Agreed, `Container` is better

Comment (Rhuagh) [reply src/aabb.rs]: Hmm, Container to me represent something that holds other objects, I.e Vec or HashMap. I do agree that the name isn't stellar

Comment (Rhuagh) [reply src/aabb.rs]: Cover? Envelope?

Comment (Rhuagh) [reply src/aabb.rs]: The the hardest things in computer science :

1. Naming things 
2. Off by one
3. I forgot the name of the third thing...

Comment (kvark) [reply src/aabb.rs]: isn't it in rust guidlines to name traits with single functions as those function? i.e. `Contains`

Comment (Rhuagh) [reply src/aabb.rs]: Can't find it in there, but that is better than the alternatives, so I'm all for it.

**Comment by Rhuagh** [comment]
Updated with new trait name.

**Comment by kvark** [comment]
Thank you!

**Pull request merged**